src/components/Themeroller/ThemeTuner/__tests__/ThemeTuner.spec.tsx

```diff
@@ -88,21 +87,60 @@ describe('ThemeTuner component', () => {
         expect(view.container).toMatchSnapshot();
     });
 
-    // TODO can't change value
-    it.skip('should invoke onChange callback when first input was changed', async () => {
+    it('should invoke onChange callback when first input was changed', async () => {
+        const onChangeMock = jest.fn();
         const user = userEvent.setup();
         const view = render(
-            <ThemeTuner config={ThemeRollerTestConfig} cssVars={{}} onChange={onChangeMock} />
+            <ThemeTuner
+                config={ThemeRollerTestConfig}
+                cssVars={{}}
+                onChange={() => onChangeMock('#ffffff')}
+            />
         );
 
-        expect(view.container).toMatchSnapshot();
-
-        const firstInput = screen.getByRole('spinbutton');
-
-        await user.type(firstInput, '#00000');
+        const itemsList = screen.getAllByRole('listitem');
+        const inputFiled = itemsList[1].children[1].children[0] as HTMLElement;
 
-        expect(onChangeMock).toHaveBeenCalledWith({
-            '--color-background': '#00000',
-        });
+        expect(view.container).toMatchSnapshot();
+        expect(itemsList[1].children[0].textContent).toBe('Foreground color');
+        expect(itemsList[1].children[1].textContent).toBe('#1d1d1b');
+
+        await inputFiled.focus();
+        await inputFiled.click();
+        await user.click(inputFiled);
+        await user.type(inputFiled, '#ffffff');
+
+        // const spinButton = screen.getByRole('spinbutton');
+        // const textBox = screen.getByRole('textbox');
+        // const listItem = screen.getAllByRole('listitem');
+        // const listBox = screen.getByRole('listbox');
+        // const tab = screen.getByRole('tab', { name: 'Colors' });
+        // const tabList = screen.getByRole('tablist');
+        //
+        // await user.click(spinButton);
+        // await user.click(textBox);
+        // await user.click(listItem[0]);
+        // await user.click(listBox);
+        // await user.click(tab);
+        // await user.click(tabList);
+
+        // wrapper
+        //     .find('ul li input')
+        //     .at(0)
+        //     .simulate('change', {
+        //         target: {
+        //             value: '#00000',
+        //         },
+        //     });
+        //
+        const ch = screen.getAllByRole('log')[1];
+        await ch.focus();
+        // await ch.click();
+        await user.click(ch);
+        await user.type(ch, '#ffffff');
```

Comment:
After get input field just hard-code role for it. But input value doesn't change